pkgs/top-level/php-packages.nix

```diff
@@ -49,12 +49,15 @@
 }:
 
 lib.makeScope pkgs.newScope (self: with self; {
-  buildPecl = import ../build-support/build-pecl.nix {
+  buildPecl = callPackage ../build-support/php/build-pecl.nix {
     php = php.unwrapped;
-    inherit lib;
-    inherit (pkgs) stdenv autoreconfHook fetchurl re2c nix-update-script;
   };
 
+  composerHooks = callPackage ../build-support/php/hooks { };
```

Comment:
When there are multiple derivations, you should use `callPackages` so that you can use `composerHooks.composerSetupHook.override …` rather than `(composerHooks.override …).composerSetupHook`.

Reply:
fixed.